replay-protected-sighash.md

```diff
@@ -70,16 +76,16 @@ Notes:
 
 #### value
 
-The 8-byte value of the amount of bitcoin spent in this input.
+The 8-byte value of the amount of Bitcoin (in satoshi) gained from this input.
```

Comment:
I'm guessing you refer to the 'in satoshi' part, which I have no issues with removing.  The other part is a bugfix.